guides/06_Meeting_Events.md

```diff
@@ -248,6 +252,22 @@ JavaScript uses the browser's [`getUserMedia` API](https://developer.mozilla.org
 
 <br>
 
+### Audio transmission related events
+
+The Javascript SDK publishes the following events to report issues with audio transmission:
+
+- The `receivingAudioDropped` event is triggered when the packet loss for receiving audio is high. This can be used to warn users about potential network connectivity issues but if you are already using the [connectionDidSuggestStopVideo](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html#connectiondidsuggeststopvideo) or [connectionDidBecomePoor](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html#connectiondidbecomepoor) callbacks you do not need to take any action as these callbacks are triggered on the same Predicate.
+
+- The `sendingAudioFailed` event is triggered when audio packets are not being sent out consistently. This event can be used to alert the user that their audio is not reaching other attendees or to ask them to take a manual action such as changing the input audio device or restarting the application. When there's a recovery in the audio packets sent, the `sendingAudioRecovered` event is published which can be used to acknowledge that the user action did succeed.  Unlike the `receivingAudioDropped` event there are no additional callbacks which are invoked for this case. There are few caveats attached to these events listed below:
+  - The `sendingAudioFailed` event may not trigger accurately in Firefox when the user switches from a working audio input device to a non-working one, this is due to a potential bug in the WebRTC's getStats() implementation within Firefox. 
+  - In certain genuine usecases where applications want to allow users to join as an observer without any audio input, we may trigger the `sendingAudioFailed` event incorrectly in Firefox and Safari. This is related to an [existing bug](https://github.com/aws/amazon-chime-sdk-js/issues/474) which actually sends such attendees into a reconnect loop and eventually results in meeting failure.
+
+
+Once published these events undergo a cooldown period to avoid inundating the event observer
```

Comment:
inundate (fun word choice though) -> overwhelm. Reducing vocabulary set/complexity improves understand-ability, especially given we aren't translating these guides :)